test/unit/membership_test.js

```diff
@@ -204,7 +191,7 @@ testRingpop('cannot evict self', function t(deps, assert) {
 testRingpop('generate checksums string preserves order of members', function t(deps, assert) {
     var membership = deps.membership;
 
-    for (var i = 0; i < 100; i++) {
+    for (var i = 1; i < 100; i++) {
```

Comment:
can you add a comment why we start? It took me some time to understand, and I hope I understand it correctly that this prevents to call `makeAlive` on the local member (?).